### PR TITLE
chore: configure base url

### PR DESCRIPTION
**Pull request by jvelezpo**
### Pre-PR Checklist

- [x] I have run `npm test` locally and there are no warnings or errors, from lint or test
- [x] I have added screenshots if applicable

### Summary

Allows to configure API url, supporting "third-party" backends like [Wakapi](https://github.com/muety/wakapi)

https://github.com/wakatime/chrome-wakatime/issues/130

Screen Shot 2023-03-15 at 3 58 53 PM

Screen Shot 2023-03-15 at 3 59 17 PM

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -118,23 +119,23 @@ const config: Config = {
...
+  currentUserApiEndPoint: process.env.CURRENT_USER_API_URL ?? '/v1/users/current',
 
   detectionIntervalInSeconds: 60,
 
   devSites:
     'https://codepen.io/\nhttps://www.codewars.com/\nhttps://dev.to/\nhttps://github.com/\nhttps://www.hackerrank.com/\nhttps://leetcode.com/\nhttps://developer.mozilla.org/en-US/\nhttps://stackoverflow.com/\nhttps://www.udemy.com/\nhttps://www.w3schools.com/',
 
-  heartbeatApiUrl:
-    process.env.HEART_BEAT_API_URL ?? 'https://wakatime.com/api/v1/users/current/heartbeats',
+  heartbeatApiEndPoint: process.env.HEART_BEAT_API_URL ?? '/v1/users/current/heartbeats',
```
Comment (gandarez) [inline src/config/config.ts:138]: Heartbeat is a single word, maybe renaming it now makes sense?

Comment (jvelezpo) [reply src/config/config.ts]: Done

**Comment by alanhamlett** [comment]
Shouldn't it be `.../api/v1` to match the wakatime-cli default? https://github.com/wakatime/wakatime-cli/blob/develop/USAGE.md#ini-config-file

**Comment by jvelezpo** [comment]
> Shouldn't it be `.../api/v1` to match the wakatime-cli default? https://github.com/wakatime/wakatime-cli/blob/develop/USAGE.md#ini-config-file

@alanhamlett now it is matching `cli` baseUrl
can you please review 👍

**Pull request merged**